test/schema/me/conversation/index.js

```diff
@@ -121,6 +123,8 @@ describe("Me", () => {
           from: {
             email: "fancy_german_person@posteo.de",
           },
+          is_last_message_to: true,
+          last_message_open: null,
```

Comment:
It _would_ be. We're several API endpoints/faffing on the Impulse + Radiation side to be able to put that in.